test/integration/answers.test.ts

```diff
@@ -0,0 +1,238 @@
+          expect(response).to.have.status(500);
+          expect(response.body.message).to.equal("An internal server error occurred");
+
+          return done();
+        });
+    });
+
+    it("should update the status with REJECTED value and set reviewed_by with rds user id if user is super user", function (done) {
+      sinon.stub(answerQuery, "updateAnswer").returns(answerDataArray[2]);
+
+      const payload: { status: AnswerStatus } = {
+        status: "REJECTED",
+      };
+
+      chai
+        .request(app)
+        .patch(`/answers/${answerDataArray[2].id}`)
+        .set("cookie", `${cookieName}=${superUserAuthToken}`)
+        .send(payload)
+        .end((error, response) => {
+          if (error) {
+            return done(error);
+          }
+          expect(response).to.have.status(204);
+
+          return done();
+        });
+    });
+
+    it("should update the status with APPROVED value and set reviewed_by with rds user id if user is super user", function (done) {
+      sinon.stub(answerQuery, "updateAnswer").returns(answerDataArray[2]);
+
+      const payload: { status: AnswerStatus } = {
+        status: "APPROVED",
+      };
+
+      chai
+        .request(app)
+        .patch(`/answers/${answerDataArray[2].id}`)
+        .set("cookie", `${cookieName}=${superUserAuthToken}`)
+        .send(payload)
+        .end((error, response) => {
+          if (error) {
+            return done(error);
+          }
+          expect(response).to.have.status(204);
+
+          return done();
+        });
+    });
+
+    it("should update the status with REJECTED value and set reviewed_by with rds user id if user is member", function (done) {
+      sinon.stub(answerQuery, "updateAnswer").returns(answerDataArray[2]);
+
+      const payload: { status: AnswerStatus } = {
+        status: "REJECTED",
+      };
+
+      chai
+        .request(app)
+        .patch(`/answers/${answerDataArray[2].id}`)
+        .set("cookie", `${cookieName}=${memberAuthToken}`)
+        .send(payload)
+        .end((error, response) => {
+          if (error) {
+            return done(error);
+          }
+          expect(response).to.have.status(204);
+
+          return done();
+        });
+    });
+
+    it("should update the status with APPROVED value and set reviewed_by to rds user id if user is member", function (done) {
+      sinon.stub(answerQuery, "updateAnswer").returns(answerDataArray[2]);
+
+      const payload: { status: AnswerStatus } = {
+        status: "APPROVED",
+      };
+
+      chai
+        .request(app)
+        .patch(`/answers/${answerDataArray[2].id}`)
+        .set("cookie", `${cookieName}=${memberAuthToken}`)
+        .send(payload)
+        .end((error, response) => {
+          if (error) {
+            return done(error);
+          }
+          expect(response).to.have.status(204);
+
+          return done();
+        });
+    });
+
+    it("should return not authorized if user is not super user", function (done) {
```

Comment:
This probably should be `not super user or member` because member can update the answer.